viewer/packages/cad-parsers/index.ts

```diff
@@ -38,7 +38,7 @@ export {
   torusLodGeometries,
   boxGeometryBoundingBox,
   quadGeometryBoundingBox,
-  torusGeometryBoundingBox,
+  /* torusGeometryBoundingBox, */
```

Comment:
All of these should be clearly commented with _why_ you've commented out the code